spec/immutable-data-structures.html

```diff
@@ -57,22 +57,46 @@ <h1>Record.fromEntries ( _iterable_ )</h1>
           <p>The parameter _iterable_ is expected to be an object that implements an @@iterator method that returns an iterator object that produces a two element array-like object whose first element is a value that will be used as a Map key and whose second element is the value to associate with that key.</p>
         </emu-note>
       </emu-clause>
-      <emu-clause id="sec-record.isrecord">
-        <h1>Record.isRecord ( _arg_ )</h1>
```

Comment:
Although `record[Symbol.toPrimitive]()` inside a try/catch block does serve as a brand check, I still prefer `Record.isRecord` since it gives a nice boolean instead of requiring exception handling. Must it be removed?

Reply:
I don't think that it *needs* to be removed, but I think the case for it is weaker. I was thinking that it would make sense to pursue Record.isRecord in the context of a more general Type.isType proposal that adds brand checks for lots of built-in types. I'd like to move forward on this more general proposal as soon as possible. Aside from establishing this more general pattern, I'm not sure why we'd have this one piecemeal thing (given that, I think we can tell programmers they can use `typeof x === "record"` normally, just like they would use for other primitive types).